### PR TITLE
Fix golint issues

### PR DESCRIPTION
**Pull request by bilal-mk**
Signed-off-by: Bilal.M.K <bilal.k@endurance.com>

Fixed the following lint issues,

Fixes https://github.com/openebs/openebs/issues/2142
Fixes https://github.com/openebs/openebs/issues/2093
Fixes https://github.com/openebs/openebs/issues/2091
Fixes https://github.com/openebs/openebs/issues/2086
Fixes https://github.com/openebs/openebs/issues/2126

### CODE REVIEW

File: cmd/maya-apiserver/app/server/http.go [hunk 1]
```diff
@@ -291,6 +293,7 @@ type HTTPCodedError interface {
 	Code() int
 }
 
+// CodedError is used to provide the Code error
```
Comment (princerachit) [inline cmd/maya-apiserver/app/server/http.go]: Can this be improved? This should talk about returning an HTTPCodedError

Comment (bilal-mk) [reply cmd/maya-apiserver/app/server/http.go]: Updated the comment

File: pkg/apis/openebs.io/v1alpha1/disk.go [hunk 2]
```diff
@@ -43,10 +43,12 @@ type DiskSpec struct {
...
 
+// DiskStatus provides current state of the disk (Active/Inactive)
 type DiskStatus struct {
-	State string `json:"state"` //current state of the disk (Active/Inactive)
+	State string `json:"state"` 
 }
 
+// DiskCapacity provides disk size in byte
 type DiskCapacity struct {
 	Storage uint64 `json:"storage"` //disk size in byte
```
Comment (princerachit) [inline pkg/apis/openebs.io/v1alpha1/disk.go]: remove the comment from here. Above comment will suffice.

**Comment by bilal-mk** [comment]
@princerachit Could you please check and merge the pull request. One integration test failed due to o% diff hit. But actually fixed lint issues

**Pull request merged**